src/components/Explore/panels/TraceViewPanelScene.tsx

```diff
@@ -155,9 +146,17 @@ const SkeletonComponent = () => {
 
 function getStyles(theme: GrafanaTheme2) {
   return {
-    container: css({
+    panelContainer: css({
+      display: 'flex',
       height: '100%',
-      width: '100%',
+
+      '& .show-on-hover': {
+        display: 'none',
+      },
```

Comment:
Ahh ok can remove this I bet if we don't add `setHoverHeader(true)` but I don't see any difference even without this style 🤔